Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/fhir/r401/BaseTest.java

```diff
@@ -831,4 +832,30 @@ public void testInclude() throws IOException {
         assertThat(qlr.getName(), equalTo("M"));
         assertThat(eqv.getOperand().get(1), instanceOf(ToConcept.class));
     }
+
+    @Test
+    public void testOverload() throws IOException {
+        CqlTranslator translator = TestUtils.runSemanticTest("fhir/r401/TestOverload.cql", 0);
```

Comment:
Suggestion: We should extend the tests to ensure that all functions are compiled, even if they aren't referenced. We now compile on-demand, so there may be dangling definitions that would "disappear" from the ELM.

Reply:
We could construct a specific test, but I definitely encountered this issue in getting past the regression tests, I initially tried to leave some functions uncompiled, but reconstructing the state necessary to do nested compilation was too big a lift and I opted for ensuring the Library is 100% compiled instead.